staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2982,6 +2982,14 @@ type PodSpec struct {
 	// Optional: Defaults to true.
 	// +optional
 	EnableServiceLinks *bool `json:"enableServiceLinks,omitempty" protobuf:"varint,30,opt,name=enableServiceLinks"`
+	// Overhead represents the resource overhead associated with running a pod for a given RuntimeClass. This field will be
+	// autopopulated at admission time by the RuntimeClass admission controller. Overhead must not be set in Pod create requests,
+	// or the request will be rejected. If RuntimeClass is configured and selected in the PodSpec, Overhead will be set
```

Comment:
Who rejects?  Is it the admission controller (I think it is?) or apiserver?  If it is admission controller we should explain that here, and detail that it CAN be disabled (which makes this comment wrong as-is).

Reply:
Indeed, the admission controller would reject.  You're right that it can be disabled - I'll update the comments to clarify this.